BingeIT/src/components/Sidebar/Sidebar.css

```diff
@@ -0,0 +1,37 @@
+.sidebar{
+  background-color:var(--bg-color2);
+  position: relative;
+  width:15%;
+  height: 100%;
+  position:fixed;
+}
+.sidebar-item{
+  display: flex;
```

Comment:
Could have used a utility class for display flex, which could save some lines of repeating code.